tests/test_project.py

```diff
@@ -763,14 +711,7 @@ def test_schema(self):
             "f.f2",
         ):
             assert k in s
-            if "." in k:
-                with pytest.warns(FutureWarning):
-                    assert k.split(".") in s
-            # The following calls should not error out.
             s[k]
```

Comment:
I think the previous comment was somewhat useful, since it's weird to see a bare `__getitem__` outside of an assignment statement or function call.
```suggestion
            # The following call should not error out.
            s[k]
```